hack/conformance.sh

```diff
@@ -11,8 +11,14 @@ mkdir -p ${ARTIFACTS}
 echo 'Obtaining KUBECONFIG of the development cluster'
 export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
 
+sonubuoy_args="--wait --plugin _out/manifests/release/conformance.yaml"
+
+if [[ ! -z "$SKIP_OUTSIDE_CONN_TESTS" ]]; then
+    sonubuoy_args="$sonubouy_args --e2e-skip \"\\[outside_connectivity\\]\""
```

Comment:
@alonSadan I got the following error when I tried to run the test using your changes (and patching the prev issues):
```code
 » make conformance SKIP_OUTSIDE_CONN_TESTS=true
hack/dockerized "export SKIP_OUTSIDE_CONN_TESTS=true && hack/conformance.sh"
selecting docker as container runtime
go version go1.16.1 linux/amd64

go version go1.16.1 linux/amd64
+ echo 'Preparing directory for artifacts'
Preparing directory for artifacts
+ export ARTIFACTS=_out/artifacts/conformance
+ ARTIFACTS=_out/artifacts/conformance
+ mkdir -p _out/artifacts/conformance
+ echo 'Obtaining KUBECONFIG of the development cluster'
Obtaining KUBECONFIG of the development cluster
++ ./cluster-up/kubeconfig.sh
+ export KUBECONFIG=/root/go/src/kubevirt.io/kubevirt/_ci-configs/k8s-1.18/.kubeconfig
+ KUBECONFIG=/root/go/src/kubevirt.io/kubevirt/_ci-configs/k8s-1.18/.kubeconfig
+ sonubuoy_args='--wait --plugin _out/manifests/release/conformance.yaml'
+ [[ ! -z true ]]
+ sonubuoy_args=' --e2e-skip "\[outside_connectivity\]"'
+ echo 'Executing conformance tests and wait for them to finish'
Executing conformance tests and wait for them to finish
+ sonobuoy --e2e-skip '"\[outside_connectivity\]"'
Error: unknown flag: --e2e-skip
Usage:
  sonobuoy [flags]
  sonobuoy [command]

Available Commands:
  delete      Deletes Kubernetes resources that were generated by a Sonobuoy run
  e2e         Inspect e2e test results. Optionally rerun failed tests
  gen         Generates a sonobuoy manifest for submission via kubectl
  help        Help about any command
  images      Manage images used in a plugin. Supported plugins are: 'e2e'
  logs        Dumps the logs of the currently running sonobuoy containers for diagnostics
  results     Inspect plugin results.
  retrieve    Retrieves the results of a sonobuoy run to a specified path
  run         Submits a sonobuoy run
  status      Gets a summarized status of a sonobuoy run
  version     Print sonobuoy version

Flags:
      --add_dir_header                   If true, adds the file directory to the header
      --alsologtostderr                  log to standard error as well as files
  -d, --debug                            Enable debug output (includes stack traces)
  -h, --help                             help for sonobuoy
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --log_file string                  If non-empty, use this log file
      --log_file_max_size uint           Defines the maximum size a log file can grow to. Unit is megabytes. If the value is 0, the maximum file size is unlimited. (default 1800)
      --logtostderr                      log to standard error instead of files (default true)
      --skip_headers                     If true, avoid header prefixes in the log messages
      --skip_log_headers                 If true, avoid headers when opening log files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          number for the log level verbosity
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

Use "sonobuoy [command] --help" for more information about a command.

ERRO[0000] unknown flag: --e2e-skip                     
make: *** [Makefile:86: conformance] Error 1
```

Reply:
I made a stupid mistake and called it `sonubuoy` instead of `sonobuoy`. Also I forgot to add run. I should have verified it better. Sorry.
now it is working.